mlflow/java/client/src/main/java/org/mlflow/artifacts/CliBasedArtifactRepository.java

```diff
@@ -167,34 +169,36 @@ private void checkMlflowAccessible() {
 
     try {
       String tag = "get mlflow version";
-      String mlflowVersion = forkProcess(Lists.newArrayList(mlflowExecutable, "--version"), tag);
```

Comment:
Unfortunately, one difference is that `--version` does not work, as the built-in support from click to implement this does not work when invoked using `python -m mlflow.cli`.